code/game/gamemodes/objective.dm

```diff
@@ -5,8 +5,11 @@ GLOBAL_LIST_INIT(potential_theft_objectives, (subtypesof(/datum/theft_objective)
 /datum/objective
 	/// Mind who owns the objective.
 	var/datum/mind/owner = null
+	//An alternative to 'owner': a team.
+	var/datum/team/team
 	/// What the owner is supposed to do to complete the objective.
 	var/explanation_text = "Nothing"
+	var/team_explanation_text //For when there are multiple owners.
```

Comment:
I don't know if you plan on using this, but currently it's unused. Surely we can just use the normal `explanation_text` though right?

Reply:
I'll look into this again as I keep going, if unneeded I'll remove it.